src/views/portfolio/projects/Project.vue

```diff
@@ -244,8 +244,6 @@
     },
     beforeMount() {
       this.uuid = this.$route.params.uuid;
-    },
-    mounted() {
```

Comment:
Are you questioning why I need to set the UUID here or why I removed the `mounted` method and moved `this.initialize` into the `beforeMount` method?

Reply:
Why removing the mounted() here and moving it

Reply:
`mounted` was accidentally removed, `this.initialize` is now in `mounted` again.
`this.uuid = this.$route.params.uuid` needs to be in `beforeMount`, otherwise the projects uuid will be null in it's detail view because of timing issues.

Reply:
Correction: `this.initialize` is necessary in `beforeMount` or else the project won't list its parent in the `ProjectDetailsModal`, unless the project page is refreshed.